UI/data/locale/en-US.ini

```diff
@@ -55,6 +55,15 @@ Enable="Enable"
 DisableOSXVSync="Disable macOS V-Sync"
 ResetOSXVSyncOnExit="Reset macOS V-Sync on Exit"
 HighResourceUsage="Encoding overloaded! Consider turning down video settings or using a faster encoding preset."
+DisableOSXVSync="Disable OSX V-Sync"
+ResetOSXVSyncOnExit="Reset OSX V-Sync on Exit"
```

Comment:
The name of OSX is now macOS.

Reply:
When rebasing, old strings got into the commit. It is fixed now.